packages/core/src/spec/lib/androidSdk/AndroidSdkToolsSpec.ts

```diff
@@ -16,7 +16,7 @@
 
 import {Config} from '../../../lib/Config';
 import {JdkHelper} from '../../../lib/jdk/JdkHelper';
-import {AndroidSdkTools} from '../../../lib/androidSdk/AndroidSdkTools';
+import {AndroidSdkTools, newAndroidSdkTools} from '../../../lib/androidSdk/AndroidSdkTools';
```

Comment:
There are unused import. That will go away one we stop exporting newAndroidSdkTools, but as a best practice `lint` will tell you that and Github highlights it too.